packages/p2p/src/components/modal-manager/modals/buy-sell-modal/buy-sell-modal.tsx

```diff
@@ -144,10 +167,12 @@ const BuySellModal = () => {
                     className={classNames('buy-sell-modal', {
                         'buy-sell-modal__form': should_show_add_payment_method_form,
                     })}
+                    has_return_icon={should_show_add_payment_method_form}
                     height={is_buy_advert ? 'auto' : '649px'}
                     is_open={is_modal_open}
+                    onReturn={onReturn}
                     portalId='modal_root'
-                    title={<BuySellModalTitle is_buy={is_buy_advert} />}
+                    title={getModalTitle()}
```

Comment:
I opted to pass a string instead of the component because this is what caused the console error issues. Because we had passed it as a component, the title prop in Modal component is wrapped inside Text which is strange as we had previously returned it as a Text Component. 

We could create another prop for specifically passing a component, but the styles are already there, and we don't need to repeat it again and style it ourselves from the child to the parent. 

There's no issue with the MobileFullPageModal, so I had left it as it is